OasisDEX/oasis-earn-sc
# Add deployment scripts for Modular MPA

Deployment instructions available [in Notion](https://www.notion.so/oazo/Modular-MPA-deployment-scripts-735c4cc5a5d541e78f6e1c2757f01704)

Steps to test locally:
1) `npx hardhat run scripts/deployment/deploy.ts --network local`
2) Add deployed contracts to /scripts/common/addresses.ts (MAINNET section for local network or Mainnet, GOERLI for testnet)
3) `npx hardhat run scripts/deployment/deploy-maker-actions.ts --network local`
4) Add deployed actions addresses to /scripts/common/addresses.ts
5) `npx hardhat run scripts/deployment/deploy-create-operations.ts --network local`
6) `npx hardhat run scripts/deployment/test-deploy.ts --network local` - this Operation should succeed and open Maker Vault for the deployer address
